### ISSUE TITLE
No data set on Dropbox

### ISSUE THREAD

**Issue by Flisa666** [abhishekrana/DeepFashion#4]
Seems that the data set is already deleted on Dropbox. It there any other way to get it?

**Comment by abhishekrana** [comment]
https://drive.google.com/drive/folders/0B7EVK8r0v71pQ2FuZ0k0QnhBQnc

**Comment by michael20at** [comment]
I downloaded this, but it is a 12 GB .h5 file, demanding a password! Are you sure this is the right link and if so what is the password? Thank you.

**Comment by abhishekrana** [comment]
The link is same as the one generated by download section of http://mmlab.ie.cuhk.edu.hk/projects/DeepFashion.html. (Maybe they have password protected the dataset now)

**Comment by michael20at** [comment]
Yes, according to their site it sadly is password protected now, you have to write them an application. Thanks for the reply though.

**Comment by rohanneps** [comment]
@michael20at Did you get a reply regarding the password?

**Comment by lqsunshine** [comment]
> @michael20at Did you get a reply regarding the password?

Did you get a reply regarding the password? the same problem
